src/nitclk/examples/nitclk_master_trigger.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import niscope
+import nitclk
+import sys
+
```

Comment:
Please update https://github.com/ni/nimi-python/blob/master/CHANGELOG.md and include it in the PR